activities/communication/how-to-post-to-social-media.md

```diff
@@ -0,0 +1,65 @@
+---
+type: guide
+explains: how to use Planable
+---
+
+# How to post to social media
+
+We use Planable to draft, review and post social media posts on Twitter, LinkedIn and Facebook. Planable is set up so that many people on the team can propose social media posts, but no posts can go live without a review.
+
+Our goals for Planable are:
+
+* to make it easy for staff to create social posts about what's interesting or exciting in their work
+* to give the communications team enough time to review social media posts without it being stressful
+* to give everyone a structured and predictable social media post creation and publishing process
+
+## Why we post to social media
+
+We use social media to share information that's interesting or exciting to us. Our information should be timely, and it's ok if posts are no longer be relevant a week after they're published.
+
+Typical content for our social posts includes:
+
+* announcements of events, blogposts, podcasts or major news
+* celebration, appreciation or acknowledgement of someone or something in our community
+* stuff we find interesting
+* things we've learned (but blog for failures, since short form posts aren't great for nuance)
+
+Most of our social followers are real people (many of whom we know!) who are already excited about public code, so we don't spam or try to convert them.
+
+[Read the commmunications plan](https://about.publiccode.net/activities/communication/communications-plan.html) for more information about our audiences and messages.
+
+## Tips for writing social media posts
+
+* The most interesting posts are the ones from our community and codebases. Try to showcase their success stories before ours.
+* Make sure the audience of your post matches the Foundation for Public Code one.
+* Keep in mind audiences in other time zones. Is your message important for them to read?
+* Use the right tone in your post. Notice that enthusiasm varies whether we're announcing something that excites us or not.
+* Tag the organizations or people you are writing about if possible.
+* Use relevant hashtags like #PublicCode,  #StandardforPublicCode, #ltapc, #OpenSource, #FOSS, #GovTech
+* In order to make the message more engaging, include an image and/or video with every post. Note that links usually unfold a preview of the content. Emojis are also welcome in moderation.
+* Always include a link for further information in case people are intrigued by what you've written.
+
+## Process
+
+* Make sure you have access to Planable. If not, ask the [communications team](https://about.publiccode.net/organization/staff.html#communications).
+* Select the social media channel/s where you want your message to be published. Note that LinkedIn's integration doesn't work great when mentioning a person or company (@____).
```

Comment:
Does this mean we should not use the integration or check if the @ mentions in LinkedIn work correctly

Reply:
It doesn't really work, so our workaround has been to edit LI posts as soon as they're live to correctly tag people. Would it be more helpful to include this instruction explicitly?

Reply:
I've updated this with more explicit instructions.